addon/babel.config.json

```diff
@@ -1,7 +1,7 @@
 {
   "presets": [["@babel/preset-typescript"]],
   "plugins": [
-    "@embroider/addon-dev/template-colocation-plugin",
+    "@babel/plugin-proposal-class-properties",
```

Comment:
@dgeb do we need to compile away class properties? I think this could be delegated to the host app but I may be wrong

Reply:
this worth separate discussion indeed. landing for now

Reply:
Good catch @SergeAstapov - this slipped in when I copied from another config. I'll PR its removal.